packages/components/src/Menu/Menu.css

```diff
@@ -91,14 +106,25 @@
   align-items: center;
 }
 
+:global(.jobber-retheme) .action {
+  gap: var(--space-small);
+  padding: var(--space-small) calc(var(--space-small) + var(--space-smaller));
+}
+
 .action:hover,
-.action:focus {
+.action:focus-visible {
   background-color: var(--color-surface--hover);
   outline-color: var(--color-focus);
 }
 
-.icon {
-  margin-right: var(--space-base);
+:global(.jobber-retheme) .action span {
+  /* can move into the Typography component post re-theme */
+  font-weight: 600;
+}
+
+:global(.jobber-retheme) .icon svg {
+  /* can move into the Icon component post re-theme */
+  fill: var(--color-interactive--subtle);
```

Comment:
This should be he darker/base blue, unless it changes to that in the theme?

Reply:
Yeah `interactive--subtle` maps to `blue` now